modules/installation-gcp-enabling-confidential-vms.adoc

```diff
@@ -18,6 +18,11 @@ You can use Confidential VMs when installing your cluster. Confidential VMs encr
 :FeatureName: Confidential Computing
 include::snippets/technology-preview.adoc[]
 
+[IMPORTANT]
+====
+Due to a known issue, you cannot create persistent volume storage on a cluster that uses Confidential VMs. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-7582[OCPBUGS-7582].
```

Comment:
PV will be created successfully but mount will fail.  So suggest update as below, wdyt?
 
```suggestion
Due to a known issue, you cannot use persistent volume storage on a cluster with Confidential VMs. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-7582[OCPBUGS-7582].
```

Reply:
Thank you for this correction.